ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-multicast.yml

```diff
@@ -2,6 +2,13 @@
 
 router_multicast:
   ipv4:
+    rpf:
+      routes:
+        - subnet: 10.10.10.1/32
```

Comment:
Sorry to bother again. I just tested these commands. I found that same subnet can have multiple nexthops.
<img width="281" alt="image" src="https://user-images.githubusercontent.com/88734745/161996131-c5c0ad63-0545-455d-b264-0c2987a75ba2.png">
<img width="281" alt="image" src="https://user-images.githubusercontent.com/88734745/161996131-c5c0ad63-0545-455d-b264-0c2987a75ba2.png">

Reply:
So instead of using 2 different knobs 1 for nexthop_ip and nexthop_interface. Better use 1 knob ```nexthop``` because in future if we wanted to support the command for vxlan or vlan or any type of supported nexthops, the current model is not scalable.

So lets use just one knob ```nexthop``` which will be a list and it will take the input as an IP or a valid interface.

Reply:
So instead of using 2 different knobs 1 for nexthop_ip and nexthop_interface. Better use 1 knob ```nexthop``` because in future if we wanted to support the command for vxlan or vlan or any type of supported nexthops, the current model is not scalable.

So lets use just one knob ```nexthop``` which will be a list and it will take the input as an IP or a valid interface.

Reply:
Suggested model: 

```yaml
rpf:
      routes:
        - subnet: < ip/mask >
         nexthops:
           - nexthop: < ip | interface_name >
             distance: < integer >
```

Also, you can come up with yours. The key names should give the user an idea.

Reply:
I am happy to change. Give me a moment.

Reply:
Thank you. But describe ```nexthop: < ip | interface_name >``` in a better way in README. That was just an example.
Use comments above that in README to give better context to the user.

Reply:
nexthops.nexthop is sort of confusing.  How about the below

```
rpf:
      routes:
        - subnet: < ip/mask >
          routes:
           - nexthop: < ip | interface_name >
             distance: < integer >
```